.github/workflows/ci.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
       - name: Check Code
         shell: powershell
         run: |
+          dotnet nuget locals all --clear
```

Comment:
IMO would be nice to have a comment explaining why we have to do this, but won't block approval on it

Reply:
I don't really know why it is required. Assume some kind of overzealous caching on those Azure machines.